packages/tidy/tidy.5.6-0.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "zandoye@gmail.com"
+authors: [ "ZAN DoYe" ]
+homepage: "https://github.com/kandu/ocaml-tidy/"
+bug-reports: "https://github.com/kandu/ocaml-tidy/issues"
+license: "MIT"
+dev-repo: "git://github.com/kandu/ocaml-tidy"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.04.2"}
+  "conf-tidy"
+  "core_kernel" {>= "v0.10.0" & < "v0.14"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.4"}
```

dune should not be tagged as build dependency due to forward compatibility issues